spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/src/main/java/org/springframework/cloud/contract/verifier/plugin/SpringCloudContractVerifierGradlePlugin.java

```diff
@@ -71,6 +77,10 @@
 
 	private static final String CONTRACT_TEST_RUNTIME_ONLY_CONFIGURATION_NAME = "contractTestRuntimeOnly";
 
+	private static final String CONTRACT_TEST_RUNTIME_CLASSPATH_CONFIGURATION_NAME = "contractTestRuntimeClasspath";
```

Comment:
Shouldn't this extend from `contractTest` ?

Reply:
So `contractTestRuntimeClasspath` is automatically created and configured to extend all `contractTest` configurations (`contractTestImplementation`, etc), so we are using this here simply as a way to gain access to that resolvable configuration that will contain all of the end users runtime classpath elements so that we can extend it one level further to re-add sc-converters back for our purposes.